include/cuco/detail/static_map.inl

```diff
@@ -422,6 +422,96 @@ __device__ bool static_map<Key, Value, Scope, Allocator>::device_mutable_view::i
   }
 }
 
+template <typename Key, typename Value, cuda::thread_scope Scope, typename Allocator>
+template <typename Hash, typename KeyEqual>
+__device__
```

Comment:
Are there plans to add a CG version of the device-side function? What about a bulk host-side API?

Reply:
We discussed a bulk-API here: https://github.com/NVIDIA/cuCollections/pull/202#issuecomment-1208131266

A CG version would be nice, but I wouldn't consider it required for this PR. It can always be added later.